pandas/tests/frame/methods/test_rename.py

```diff
@@ -173,7 +173,7 @@ def test_rename_multiindex(self):
     @td.skip_array_manager_not_yet_implemented  # TODO(ArrayManager) setitem copy/view
     def test_rename_nocopy(self, float_frame):
         renamed = float_frame.rename(columns={"C": "foo"}, copy=False)
-        renamed["foo"] = 1.0
+        renamed["foo"][:] = 1.0
```

Comment:
Is this needed? Testing it out with this branch seems to still change the original `float_frame` with `renamed["foo"] = 1.0` as well

Reply:
Ah, no, I tested it with a non-renamed columns .. Setting "foo" indeed no longer changes the original frame.